packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -26,6 +26,17 @@ div:has(.#{$prefix}--header)
   inset-block-start: $spacing-09;
 }
 
+.#{$prefix}--side-nav--expanded:not(.#{$prefix}--side-nav--ux) {
+  z-index: 9999;
+  inset-block-start: 0;
+  block-size: 100%;
+}
+
+.#{$prefix}--side-nav__overlay-hover.#{$prefix}--side-nav__overlay-active {
+  inset-block-start: 0;
+  z-index: 9999;
+}
+
```

Comment:
```suggestion
```

So I think we can remove all of this

Reply:
oh ok so the expanded local nav doesn't become full height?